api/envoy/config/filter/http/aws_lambda/v2/aws_lambda.proto

```diff
@@ -101,6 +101,8 @@ message AWSLambdaConfig {
     string uri = 2 [ (validate.rules).string.min_bytes = 1 ];
     // timeout for the request
     google.protobuf.Duration timeout = 3;
+    // Region for the sts endpoint, defaults to us-east-1
```

Comment:
```suggestion
    // Region for the sts endpoint, defaults to us-east-1. 
    // This must be an enabled region https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html
    // If the caller lives in an aws region they should use the same endpoint.
```

Reply:
Updated here, let me know what you think: https://github.com/solo-io/envoy-gloo/pull/267/commits/571ad3930983ba28214295adc54467a8396cccb2

```cpp
    // Region for the sts endpoint, defaults to us-east-1. 
    // This must be an enabled region https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html
    // This should match the region specified in the uri.
```